types/prettier/doc.d.ts

```diff
@@ -81,26 +101,62 @@ export namespace builders {
         placeholder: symbol;
     }
 
+    interface GroupOptions {
+        shouldBreak?: boolean;
+        id?: symbol;
+    }
+
     function addAlignmentToDoc(doc: Doc, size: number, tabWidth: number): Doc;
```

Comment:
@fisker I couldn't find this in the `commands.md` :thinking: 
Is this completely removed? Or just undocumented?

Reply:
Missing

Reply:
Okay so I will already add the link and you can update the `command.md` on your side? :slightly_smiling_face:

Reply:
https://github.com/prettier/prettier/issues/10962